data/content/environment_variables.yaml

```
@@ -269,7 +269,7 @@ variables:
     - false
 - name: BUILDKITE_MESSAGE
   desc: |
-    The message associated with the build, usually the commit message.
+    The message associated with the build, usually the commit message. This value can be blank if message is not set when builds manually triggered using Buildkite web interface.
```

I think splitting up the ideas a bit makes it simpler to understand:

```suggestion
    The message associated with the build, usually the commit message. The value is empty when a message is not set. For example, when a user triggers a build from the Buildkite dashboard without entering a message, the variable returns an empty value.
```

What do you think?

---

Should we say Buildkite dashboard or Buildkite web interface ? I saw for few of the variable we say Buildkite web interface so just confirming.

---

I found a post that pre-dates me about trying to move towards _Buildkite dashboard_, so have been using that for new content or updates.

---

Thanks for confirming. I made the changes suggested